### PR TITLE
Update dynamic filtering stats documentation

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

File: docs/src/main/sphinx/admin/dynamic-filtering.rst [hunk 1]
```diff
@@ -157,6 +155,36 @@ processed after a dynamic filter is pushed down to the table scan.
...
+        │      Layout: [ss_sold_date_sk:bigint, $hashvalue:bigint]
+        │      Estimates: {rows: 120527 (2.03MB), cpu: 1017.64k, memory: 0B, network: 0B}/{rows: 120527 (2.03MB), cpu: 1.99M, memory: 0B, network: 0B}/{rows: 120527 (2.03MB), cpu: 4.02M, memory: 0B, network: 0B}
+        │      CPU: 49.00ms (18.85%), Scheduled: 123.00ms (19.62%), Output: 120527 rows (2.07MB)
+        │      Input avg.: 120527.00 rows, Input std.dev.: 0.00%
+        │      $hashvalue := combine_hash(bigint '0', COALESCE("$operator$hash_code"("ss_sold_date_sk"), 0))
+        │      ss_sold_date_sk := ss_sold_date_sk:bigint:REGULAR
+        │      Input: 120527 rows (1.03MB), Filtered: 0.00%
+        │      Dynamic filters:
+        │          - df_370, [ SortedRangeSet[type=bigint, ranges=3, {[2451546], ..., [2451905]}] ], collection time=2.34s
+
```
Comment (sopel39) [inline docs/src/main/sphinx/admin/dynamic-filtering.rst:185]: you could add remaining `ScanFilterProject` lines too

**Review by sopel39 — APPROVED** [review]
@mosabua please review

File: docs/src/main/sphinx/admin/dynamic-filtering.rst [hunk 2]
```diff
@@ -157,6 +155,36 @@ processed after a dynamic filter is pushed down to the table scan.
     "inputPositions" : 28800991,
     "dynamicFilterSplitsProcessed" : 1,
 
+Dynamic filters collected by the coordinator are also reported as a part of the
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/dynamic-filtering.rst:158]: drop "collected by the coordinator" and "also"

**Review by mosabua — CHANGES_REQUESTED** [review]

File: docs/src/main/sphinx/admin/dynamic-filtering.rst (hunk 1 above)
Comment (raunaqmorarka) [reply docs/src/main/sphinx/admin/dynamic-filtering.rst:185]: That is the end of output for ScanFilterProject, subsequent lines are for other parts of the plan

File: docs/src/main/sphinx/admin/dynamic-filtering.rst (hunk 2 above)
Comment (raunaqmorarka) [reply docs/src/main/sphinx/admin/dynamic-filtering.rst]: done

**Review by mosabua — APPROVED** [review]

File: docs/src/main/sphinx/admin/dynamic-filtering.rst [hunk 3]
```diff
@@ -129,9 +129,7 @@ by the coordinator can be found in the ``dynamicFiltersStats`` structure.
     "dynamicFiltersStats" : {
           "dynamicFilterDomainStats" : [ {
             "dynamicFilterId" : "df_370",
-            "simplifiedDomain" : "[ [[2451546, 2451905]] ]",
-            "rangeCount" : 3,
-            "discreteValuesCount" : 0,
+            "simplifiedDomain" : "[ SortedRangeSet[type=bigint, ranges=3, {[2451546], ..., [2451905]}] ]",
```
Comment (m57lyra) [inline docs/src/main/sphinx/admin/dynamic-filtering.rst:132]: i feel this should be formatted one item per line for readability

**Review by m57lyra — APPROVED** [review]

File: docs/src/main/sphinx/admin/dynamic-filtering.rst (hunk 3 above)
Comment (raunaqmorarka) [reply docs/src/main/sphinx/admin/dynamic-filtering.rst:132]: this is a copy-paste from actual query's queryinfo json

File: docs/src/main/sphinx/admin/dynamic-filtering.rst [hunk 4]
```diff
@@ -157,6 +155,36 @@ processed after a dynamic filter is pushed down to the table scan.
     "inputPositions" : 28800991,
     "dynamicFilterSplitsProcessed" : 1,
 
+Dynamic filters are reported as a part of the
+:doc:`EXPLAIN ANALYZE plan </sql/explain-analyze>` in the statistics for
```
Comment (rosewms) [inline docs/src/main/sphinx/admin/dynamic-filtering.rst:159]: If I'm being picky, "Dynamic filters report as a part of..." reads better.

**Review by electrum — APPROVED** [review]

**Pull request merged**